components/search_results/index.tsx

```diff
@@ -23,6 +22,9 @@ import {
 import {GlobalState} from 'types/store';
 import {Preferences} from 'utils/constants';
 
+import {FileSearchResultItem} from '@mattermost/types/files';
+import {Post} from '@mattermost/types/posts';
```

Comment:
Could you revert these cosmetic changes from the PR? Since this is going in for 7.4, those extra changes might make cherry-picking either this PR or future ones more difficult because of potential conflicts

Reply:
Will do, but those are showing as eslint errors for me. Did anyone roll some of the rules?

Reply:
I did change it back, but it still gave me errors ... I re-structured the imports to be more logical again, so feel free to tag me for merging this PR commit when we do the cherry-picking